Source/IGListAdapter.m

```diff
@@ -414,6 +414,23 @@ - (NSArray *)visibleObjects {
     return [visibleObjects allObjects];
 }
 
+- (NSArray<UICollectionViewCell *> *)visibleCellsForObject:(id)item {
+    IGAssertMainThread();
+    IGParameterAssert(item != nil);
+    
+    NSArray<UICollectionViewCell *> *visibleCells = [self.collectionView visibleCells];
+    NSInteger section = [self.sectionMap sectionForObject:item];
+    IGAssert(section != NSNotFound, @"Section not found for object %@", item);
+    
+    NSPredicate *controllerPredicate = [NSPredicate predicateWithBlock:^BOOL(UICollectionViewCell* cell, NSDictionary* bindings) {
+        NSIndexPath *indexPath = [self.collectionView indexPathForCell:cell];
```

Comment:
nit: calling `self.collectionView` inside this block will execute an extra `objc_msgSend(...)` every execution. Instead let's capture the collection view in a local variable that the block uses:
```objc
UICollectionView *collectionView = self.collectionView;
NSPredicate *predicate = [... ^{
  NSIndexPath *path = [collectionView indexPathForCell:cell];
}];
```